### ISSUE TITLE
List Items Checkboxes are not accessible

### ISSUE THREAD

**Issue by coliff** [trimble-oss/website-modus-bootstrap.trimble.com#139]
### Describe the issue

In the example: https://modus-bootstrap.trimble.com/components/lists/#list-items the checkboxes do not have a label.
They can't be added with CSS as it'd break the formatting. There isn't a clear workaround here so I suggest we mention this in the docs.

### What operating system(s) are you seeing the problem on?

Windows

### What browser(s) are you seeing the problem on?

Microsoft Edge

**Comment by coliff** [comment]
UPDATE: I found a fix for this 
https://modus-bootstrap.netlify.app/docs/5.2/components/list-group/#checkboxes-and-radios
